server-environment/server-environment.md

```diff
@@ -7,15 +7,24 @@ PHP `7.0` or greater is highly encouraged but WordPress will run, with slower pe
 ## PHP Extensions
 WordPress core makes use of PHP extensions. If the the preferred extension is missing WordPress will either have to do more work to do the task the module helps with or, in the worst case, will remove functionality. Therefore the PHP extensions listed below are recommended.
 
-* filter - Used for securely filtering user input
+* bcmath - Used to improve the performance of math calculations.
+* curl - Performs remote request operations.
+* filter - Used for securely filtering user input.
 * fileinfo - Used to detect mimetype of file uploads
-* xml - Used for XML parsing, such as from a third-party site 
-* pcre - Increases performance of pattern matching in code searches
-* mod_xml - Used for generating XML, such as for an XML sitemap
-* bcmath - Used to improve the performance of math calculations
+* mod_xml - Used for generating XML, such as for an XML sitemap.
+* mysqli - Connects to MySQL for database interactions.
+* libsodium - Generates random bytes.
+* openssl - Permits SSL-based connections to other hosts.
+* pcre - Increases performance of pattern matching in code searches.
 * imagick - Provides better image quality for media uploads. See [WP_Image_Editor is incoming!](https://make.wordpress.org/core/2012/12/06/wp_image_editor-is-incoming/) for details. Smarter image resizing (for smaller images) and PDF thumbnail support, when Ghost Script is also available.
+* xml - Used for XML parsing, such as from a third-party site.
 
-For the sake of completeness bleow is a list of the remaining PHP modules WordPress _may_ use in certain situations or if other modules are unavailable. These are fallbacks and not needed in an optimal environment, but installing them won't hurt.
+For the sake of completeness, below is a list of the remaining PHP modules WordPress _may_ use in certain situations or if other modules are unavailable. These are fallbacks and not needed in an optimal environment, but installing them won't hurt.
+
+* gd - If Imagick isn't installed, the GD Graphics Library is used as an alternative for image manipulation.
```

Comment:
I'd mention that this is a functionally limited fallback, rather than an alternative.